### PR TITLE
W-15924518-Add-Hanna-comments-to-MUunit-in-Studio-docs-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/coverage-studio-concept.adoc [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
 
-The following configurations apply when you execute your MUnit tests in Studio. They don't apply when running tests from Maven in a CI/CD environment. 
-
-To run coverage reports from Maven, see xref:munit::coverage-maven-concept.adoc[].
+The following configurations apply when you execute your MUnit tests in Studio. They don't apply when running tests from Maven in a CI/CD environment. To run coverage reports from Maven, see xref:munit::coverage-maven-concept.adoc[].
 
 == Enable Coverage Reports in Anypoint Studio
 
-See which flows, and which percentage of event processors in those flows the test covers by clicking on the Coverage button:
+To see which flows, and which percentage of event processors in those flows the test covers, click on the Coverage button:
```
Comment (hannanelson) [inline modules/ROOT/pages/coverage-studio-concept.adoc]:
```suggestion
To see which flows, and which percentage of event processors in those flows the test covers, click the *Coverage* button:
```

File: modules/ROOT/pages/coverage-studio-concept.adoc [hunk 2]
```diff
@@ -1,35 +1,39 @@
...
+| Column | Description
+
+| *Resource* 
+| Name of the Mule configuration file
+
+| *Containers* 
+| Number of flows in the Mule configuration file
+
+| *Weight* 
+| Percentage of Mule configuration file relative to the entire application. The percentage represents the proportion of event processors within your file compared to the total number of event processors across all configuration files
```
Comment (hannanelson) [inline modules/ROOT/pages/coverage-studio-concept.adoc]: The first sentence is confusing to me.

Comment (hannanelson) [inline modules/ROOT/pages/coverage-studio-concept.adoc]:
```suggestion
| Proportion of event processors within the Mule configuration file compared to the total number of event processors across all configuration files
```

Comment (hannanelson) [reply modules/ROOT/pages/coverage-studio-concept.adoc]: This suggestion also gets rid of the need for the period in the first sentence and the lack of one in the second - and keeps the descriptions parallel (all sentence fragments). Something to consider if you need to reword.

File: modules/ROOT/pages/coverage-studio-concept.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/coverage-studio-concept.adoc]: OK! Thanks!

File: modules/ROOT/pages/coverage-studio-concept.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/coverage-studio-concept.adoc]: OK! Thanks!

Comment (Crispy-Salesforce) [reply modules/ROOT/pages/coverage-studio-concept.adoc]: Resolved with your suggestion.
Thanks!

**Review by Cristian-Venticinque — APPROVED** [review]

**Pull request merged**